setup.cfg

```diff
@@ -1,3 +1,13 @@
 [flake8]
 max-line-length = 100
-ignore = N806, N803, N802, I100, I101, I201
\ No newline at end of file
+ignore = N806, N803, N802, I100, I101, I201
+
+[aliases]
+test=pytest
+
+[tool:pytest]
+testpaths = mdbenchmark/tests
+addopts = --cov=./
+
+[coverage:run]
+omit = mdbenchmark/tests/*
```

Comment:
we want to know if our tests are all run!

Reply:
Isn't that the job of `pytest`? It will give you a list of tests that ran and those that were skipped/failed..?